Basic/APIDiorama/Packages/manifest.json

```diff
@@ -0,0 +1,54 @@
+{
+  "dependencies": {
+    "com.unity.2d.sprite": "1.0.0",
+    "com.unity.cinemachine": "2.8.9",
+    "com.unity.feature.development": "1.0.1",
+    "com.unity.ide.rider": "3.0.15",
+    "com.unity.ide.visualstudio": "2.0.16",
+    "com.unity.ide.vscode": "1.2.5",
+    "com.unity.inputsystem": "1.4.4",
+    "com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop",
+    "com.unity.multiplayer.tools": "1.0.0",
+    "com.unity.netcode.gameobjects": "1.2.0",
+    "com.unity.render-pipelines.universal": "12.1.7",
+    "com.unity.test-framework": "1.1.31",
+    "com.unity.textmeshpro": "3.0.6",
+    "com.unity.timeline": "1.6.4",
+    "com.unity.ugui": "1.0.0",
+    "com.veriorpies.parrelsync": "https://github.com/VeriorPies/ParrelSync.git?path=/ParrelSync#6be2a26245e29070bf589809f10637af43570050",
+    "devlocker.utils.scenereference": "https://github.com/NibbleByte/UnitySceneReference.git#upm",
```

Comment:
just tested real quick, you can use https://docs.unity3d.com/ScriptReference/SceneAsset.html SceneAsset and then use "name" to get the scene name. No need for a 3rd party.

Reply:
SceneAsset is derived from UnityEditor so you can't make builds with that IIRC.

Reply:
Do we care about builds in our case? Users should use ParrelSync

Reply:
We definitely care about builds as our users care, so this is staying as it is

Reply:
reopening for tracking purposes until internal conversation is done

Reply:
Package removed, we're now using SceneAsset in Editor classes and plain strings in runtime classes